The strftime %Z format specifier returns UTC on musl in this
context, force use of GMT to prevent client errors.
See https://gitlab.alpinelinux.org/alpine/aports/-/issues/12979
--- src/althttpd.c.orig
+++ src/althttpd.c
@@ -617,7 +617,7 @@
   struct tm *tm;
   static char zDate[100];
   tm = gmtime(&t);
-  strftime(zDate, sizeof(zDate), "%a, %d %b %Y %H:%M:%S %Z", tm);
+  strftime(zDate, sizeof(zDate), "%a, %d %b %Y %H:%M:%S GMT", tm);
   return zDate;
 }
 
